synapse/federation/sender/per_destination_queue.py

```diff
@@ -138,7 +155,11 @@ def send_pdu(self, pdu: EventBase) -> None:
         Args:
             pdu: pdu to send
         """
-        self._pending_pdus.append(pdu)
+        if not self._catching_up:
```

Comment:
shouldn't we update `_catchup_last_skipped` here if the event is dropped?

Reply:
Oops, yes… odd, must have imagined putting that in